mlflow/xgboost.py

```diff
@@ -416,10 +443,51 @@ def log_feature_importance_plot(features, importance, importance_type):
             finally:
                 shutil.rmtree(tmpdir)
 
-        try_mlflow_log(log_model, model, artifact_path="model")
+        # dtrain must exist as the original train function already ran successfully
+        dtrain = args[1] if len(args) > 1 else kwargs.get("dtrain")
+
+        input_example_info = None
```

Comment:
I think this line is unnecessary, since `input_example_info` is always reassigned in the `try` block to the output of `getattr(...)`.